mmedit/datasets/pipelines/matting_aug.py

```diff
@@ -555,3 +555,48 @@ def __repr__(self):
                      f'dilate_iter_range={self.dilate_iter_range}, '
                      f'blur_ksizes={self.blur_ksizes})')
         return repr_str
+
+
+@PIPELINES.register_module()
+class TransformTrimap(object):
+    """Generate two-channel transformed trimap.
+
+    Required key is "trimap", added key is "trimap_o" and "transformed_trimap".
+
+    Adopted from the following repository:
+    https://github.com/MarcoForte/FBA_Matting/blob/master/networks/transforms.py.
+
+    """
+
+    def __call__(self, results):
+        """Call function.
+
+        Args:
+            results (dict): A dict containing the necessary information and
+                data for augmentation.
+
+        Returns:
+            dict: A dict containing the processed data and information.
+        """
+        # generate two channel trimap
+        trimap_o = results['trimap']
```

Comment:
variable name:
trimap_o -> trimap, for the original trimap
trimap -> trimap2, for 2 channel trimap